xml-isogen.cabal

```diff
@@ -31,6 +31,7 @@ library
   exposed-modules:     Data.THGen.Enum
                      , Data.THGen.XML
                      , Data.THGen.XML.Internal
+                     , Data.THGen.Nillable
```

Comment:
I feel like `Nillable` is more general than `THGen`. I mean it's not related to TH in any way. Maybe `Data.XML.Nillable`?

Reply:
Good point. I moved it to `Text.XML.Nillable`